### ISSUE TITLE
[BUG]: ModuleNotFoundError: No module named 'streamdeck_ui'

### ISSUE THREAD

**Issue by dhtseany** [streamdeck-linux-gui/streamdeck-linux-gui#208]
### In raising this issue I confirm that

- [X] I have fully completed the issue template
- [X] I have searched open and closed issues for duplicates
- [X] I have read the [Contribution Guidelines](https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/contributing/contributing-guide/)
- [X] I have read the [Code of Conduct](https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/contributing/code-of-conduct/)
- [X] I have read the [Documentation](https://streamdeck-linux-gui.github.io/streamdeck-linux-gui/)

### Describe the bug

On Arch Linux recent updates are now producing the following missing module error message:
$ streamdeck
Traceback (most recent call last):
  File "/usr/bin/streamdeck", line 5, in <module>
    from streamdeck_ui.gui import start
ModuleNotFoundError: No module named 'streamdeck_ui'

### Steps to reproduce the behavior

Launch from terminal.

### Expected behavior

Regular launch,

### System Information

Arch Linux Current - 2024.04.01
Python 3.12.3

### Stream Deck Version

Any

**Comment by coolapso** [comment]
I wont be available to take a look at this anytime soon, but if someone can make a PR with the fix I can look at it. 

In the meantime I highly advise users to sart moving towards streamcontroller. 

https://github.com/streamcontroller/streamcontroller

**Comment by dhtseany** [comment]
Hello Carlos, as you know I'm the streamdeck-ui package maintainer on Arch and while I agree that we should all be looking towards eventually moving to StreamController it's still considered beta and isn't yet production ready to replace streamdeck-ui.

That said, this isn't a feature request, it's a request for a bug fix for something that's probably relatively basic to correct and I'd hope it'd get looked at sooner than later as we were told streamdeck-ui would continue receiving basic bug-fixes for the foreseeable future, at least until StreamController releases it's first v1 non-beta.

**Comment by dhtseany** [comment]
Problem was resolved by performing a clean build of both python-elgato-streamdeck and streamdeck-ui.

**Comment by coolapso** [comment]
@dhtseany yes, that is correct. and this issue falls right under the things that should be fixed until stramcontroller is up to shape. I just cannot look into it at the moment as I am currently away, only with github on my phone and no streamdeck. 

however if someone has a fix and a PR to submit  I can gladly take a look and approve :)

**Comment by dhtseany** [comment]
@coolapso I apologize for my tone, after re-reading my first response I realized that I came off sounding too strong. As stated in my closing comment I figured out with the help of other Arch users that the 2 packages referenced simply needed rebuilt and it was an Arch-specific problem that I should've caught on my own. Regardless of outcome a friendlier demeanor is always appropriate when working with fellow volunteers and sorry for sounding harsh. Thanks for your efforts, I'll be cool! :)
